web-console/knikubevirt/Create-vm/step-4-advanced/wizard-advanced.md

```diff
@@ -0,0 +1,30 @@
+# Create A Virtual Machine
+
+## Step 4: Advanced
+
+The “Advanced” setting step of the create VM wizard includes:
+
+- Cloud-Init / Sysprep - determines the contents of these settings based on the OS that is selected. (The difference between the cloud-init and the sys-prep is basically the difference between windows and other OSs)
+- Virtual hardware
+
+### Cloud-Init / Sysprep
+
+The user chooses the "Use cloud-init/Sysprep" checkbox at the top of the form and sees the settings.
+
+- SSH Authorized Keys: Enter a comma-separated list of public SSH keys that can be used for authentication.
+- NIC: Select the NIC that should communicate with the DNS servers.
+- DNS Servers: Enter a comma-separated list of DNS server IP addresses.
+- DNS Search Domains: Enter a comma-separated list of DNS search domain IP addresses.
+
+![empty form](img/form-empty.png)
```

Comment:
I think the first check box should simply be "Use cloud-init/Sysprep". I'm not sure we really need the extra -"on" or "off" at the end since it's implied with the checkbox.